src/DynamoCoreWpf/Views/Core/DynamoView.xaml

```diff
@@ -724,6 +724,55 @@
                                   IsCheckable="True"
                                   IsChecked="{Binding ShowTabsAndSpacesInScriptEditor, Mode=TwoWay}"
                                   Header="{x:Static p:Resources.DynamoViewSettingsMenuShowTabsAndSpacesInScriptEditor}"/>
+                        <MenuItem Focusable="False"
+                                  Header="{x:Static p:Resources.DynamoViewSettingsDefaultPythonEngine}">
+                            <ComboBox ItemsSource="{Binding DefaultPythonEngineOptions}" SelectedValue="{Binding DefaultPythonEngine, Mode=TwoWay}" SelectedValuePath="Value" DisplayMemberPath="Name"/>
+                        </MenuItem>
```

Comment:
Used a drop down. As you can see from the commented markup I also tried child checkable menu items, which had some limitations. Open to discussion.

Reply:
@mmisol Looking back on this, what limitations are in this case? Is it because we want to be dynamic on this and not achievable using child menu items?